packages/esm-patient-chart-app/src/visit/start-visit-button.component.tsx

```diff
@@ -6,17 +6,23 @@ import { showSnackbar } from '@openmrs/esm-framework';
 
 interface StartVisitButtonProps {
   patientUuid: string;
+  handleBackToSearchList?: () => void;
+  setIsPatientSearchOpen?: (isOpen: boolean) => void;
```

Comment:
I'm afraid this is too specific which introduces some kind of tight coupling, what do you think of using a callback instead? Something like `onCallStartVisit` or `onStartVisit`.